src/screens/ChatScreen/index.js

```diff
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import styles from './styles';
 import { GiftedChat, Composer, InputToolbar } from 'react-native-gifted-chat';
-import AssetsImages from '../../global/AssetsImages';
+import AssetsImages from '../../global/assetsImagesResources';
```

Comment:
I'm not sure if naming is a good one. If there is a reason why it should not be just 'assets' or just 'images', please post it here in comment. Thanks!